docs/README.md

```diff
@@ -186,3 +187,9 @@ Remotely debugging the Mona web app using the method described above requires th
 ## Does Mona have a health check endpoint?
 
 Yes it does. Mona exposes a public health check `HTTP GET` endpoint at `/health`. If the Mona deployment is healthy and able to access all of its service dependencies, the endpoint will return a `HTTP 200 OK`. Any other status code indicates that Mona is unhealthy. [The Mona upgrade script uses the endpoint to ensure that the deployment is healthy before committing the upgrade to production.](#how-do-i-upgrade-mona) You can also use this health check endpoint to deploy Mona in a highly available configuration using [Azure Front Door](https://docs.microsoft.com/azure/frontdoor/health-probes), [Application Gateway](https://docs.microsoft.com/azure/application-gateway/application-gateway-probe-overview), [Kubernetes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/), etc. [You can also leverage the Application Insights resource deployed with Mona to configure availability tests using the health check endpoint.](https://docs.microsoft.com/azure/azure-monitor/app/availability-standard-tests)
+
+## After completing the purchase, the subscription in Marketplace is not active. How to fix that?
+
+It's due to the fact that `mona-on-subscription-purchased-DEPLOYMENT_NAME` Logic App does not notify the Microsoft Marketplace that subscription is activated. You can change that behaviour by changing the settings in `Notify the Marketplace` step in this Logic App.
```

Comment:
Recommend -- "It's due to the fact that __the__ `mona-on-subscription-purchased-DEPLOYMENT_NAME` Logic App does not notify the Microsoft Marketplace that subscription is activated __by default__."